clients/web/src/templates/body/systemConfiguration.jade

```diff
@@ -67,7 +67,9 @@ form.g-settings-form(role="form")
               placeholder="Default: #{defaults['core.upload_minimum_chunk_size'] || ''}",
               title="For large files, the minimum size of all but the last chunk.")
           | CORS (Cross-origin resource sharing) allows access from other
-          | websites.
+          | websites. These settings are described in detail
+          a(href="http://girder.readthedocs.org/en/latest/security.html#cors-cross-origin-resource-sharing"
```

Comment:
Will `href="security.html#cors-cross-origin-resource-sharing"` link properly, rather than using the full link which includes `http://girder.readthedocs.org`?